src/components/code-editor/code-editor.tsx

```diff
@@ -11,6 +11,7 @@ import {
 import { ColorScheme, Language } from '../../interface';
 import CodeMirror from 'codemirror';
 import 'codemirror/mode/javascript/javascript';
+import 'codemirror/mode/jinja2/jinja2';
```

Comment:
We weren't getting any syntax highlighting, so I added this.

Reply:
Oh, I got in crm-components but maybe it just discarded this change in lime-elements 😅